core/src/main/java/org/apache/calcite/prepare/CalciteCatalogReader.java

```diff
@@ -283,7 +283,7 @@ private static SqlMonikerImpl moniker(CalciteSchema schema, @Nullable String nam
    * or classes.
    *
    * @see ModelHandler#addFunctions */
-  public static SqlOperatorTable operatorTable(String... classNames) {
+  public SqlOperatorTable operatorTable(String... classNames) {
```

Comment:
why remove the static?

Reply:
The operatorTable method will call the toOp method. If the toOp method cannot be a static method, then the operatorTable method also needs to remove static.

Reply:
Besides, I found operatorTable is not called anywhere, should we delete this method?

Reply:
See my comment below about public methods; you cannot change public method signature without changing the major version of the project. That is called "semantic versioning" https://semver.org/